### ISSUE TITLE
IITC-CE Mobile not loading external plug-ins.

### ISSUE THREAD

**Issue by fortepc** [IITC-CE/ingress-intel-total-conversion#136]
It's me again hi. 

So, I'm having an issue trying to load external plugins. It doesn't work for a number of plugins that I've tried. And it's also not working in the iitc.me fork. They are plugins that I know work just fine on desktop, and I at first reported it to the plugin developer of a particular plugin assuming that it was an issue with the plug in. It's a plug-in called total recon from the same repo that I reported an issue about before. Anyway, the developer doesn't seem to think of an issue with his plug in and now that I've noticed that it's not working for a few different plugins I'm starting to think that it might be an issue with something on my phone specifically, or something with the app itself.

I'm not sure how something on my phone specifically could be causing the issue but just for completeness I'm running a Google pixel on Android P, build number PQ2A.190305.002. 

Here is the link to the issue on the other repo: https://github.com/Wintervorst/iitc/issues/2

**Comment by johnd0e** [comment]
> Here is the link to the issue on the other repo: [Wintervorst/iitc#2](https://github.com/Wintervorst/iitc/issues/2)

Answered there.

**Comment by fortepc** [comment]
Okay, sorry for the late responce, but that answer you posted there didn't work for me I wanted to do a bit more diagnostics on my side because things were being inconsistent and weird. 

So, first, that link you provided in that thread just opens up the file in the browser on most of my devices, the Pixel XL I made the report about initially did the exact same thing as I reported earlier. It let  the IITC app open the link, but then it showed up in the external plugins as "unknown". On all my other devices it would just open the link in the chrome browser and show the plugin as text. 

What ended up working was installing ES File Explorer, then going to the IITC app's plugin settings, importing the plugin, selecting ES File Explorer on the stage left to do the import of the plugin, then selecting the plugin, then doing it the "File Way". Something about the stock android file picker wasn't working. It didn't even show the plugins as valid clickable files if I tried to use the stock android file picker.

This method worked on all of my devices so far.
